extensions/pkg/controller/heartbeat/cmd/options.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	controllercmd "github.com/gardener/gardener/extensions/pkg/controller/cmd"
+	"github.com/gardener/gardener/extensions/pkg/controller/heartbeat"
+	"github.com/spf13/pflag"
+)
+
+// Options are command line options that can be set for the heartbeat controller.
+type Options struct {
+	controllercmd.ControllerOptions
+	// ExtensionName is the name of the extension controller.
+	ExtensionName string
+	// Namespace is the namespace which will be used for the heart beat lease resource.
+	Namespace string
+	// RenewIntervalSeconds defines how often the heart beat lease is renewed.
+	RenewIntervalSeconds int32
+
+	config *Config
+}
+
+// AddFlags implements Flagger.AddFlags.
+func (c *Options) AddFlags(fs *pflag.FlagSet) {
+	c.ControllerOptions.AddFlags(fs)
+	fs.StringVar(&c.Namespace, "namespace", c.Namespace, "The namespace to use for the heart beat lease resource.")
+	fs.Int32Var(&c.RenewIntervalSeconds, "renew-interval-seconds", c.RenewIntervalSeconds, "How often the heart beat lease will be renewed. Default is 30 seconds.")
```

Comment:
Why not make this a `time.Duration`?

Reply:
That's how I did it initially, but then I noticed that for the heartbeat of `gardenlet` we use int32. It's the same for the node leases. So for consistency i decided to also make this int32.